### ISSUE TITLE
Make time interval configuration easier

### ISSUE THREAD

**Issue by hsch** [mfbieber/PomodoroTimer#1]
It's really hard to use the sliders and hit a precise setting, e.g. exactly 5 minutes.

Ideas:
- Make the sliders use the full width of the screen (they are unnecessarily tiny at the moment)
- Use a textbox instead
- Just provide 4 or 5 reasonable options, e.g. "10", "15", "25", "55" or so.

**Comment by mfbieber** [comment]
Yep, I know that. I will try to make the sliders bigger or use a different thing instead!
